tsconfig.json

```diff
@@ -16,8 +16,27 @@
       "display-node_utils": ["./src/display/node_utils"],
       "fluent-bundle": ["./node_modules/@fluent/bundle/esm/index.js"],
       "fluent-dom": ["./node_modules/@fluent/dom/esm/index.js"],
-      "web-null_l10n": ["../web/genericl10n.js"]
+
```

Comment:
@Snuffleupagus should I get rid of path aliases as well? Those have no use besides generating types, but was hard to figure out the list so could be helpful for someone else like myself.

Reply:
Yes, given that we don't want to add unused code.

(Those shouldn't need to be figured out, since they can basically be copied from `gulpfile.mjs`.)